scss/_patterns_footer.scss

```diff
@@ -42,7 +48,6 @@
     &__link {
       border-bottom: 0;
       color: $color-dark;
```

Comment:
By not setting a color here, they will be the default link color, which is blue.